test/index.js

```diff
@@ -13,6 +13,7 @@ const folder = 'buckets'
 const bucket = 'S3Lambda'
 const prefix = 'files'
 const outputPrefix = 'output-files'
+// const files = ['file1', 'file2', 'file3', 'file4']
```

Comment:
it's good to get in the habit of interactively staging before commits with
```
git add -p
```
then
```
git commit -m "message"
```